SECURITY.md

```diff
@@ -0,0 +1,67 @@
+# Security
+
+## Reporting a Vulnerability
+
+If you think you have found a potential security vulnerability in `checkov`,
+please email info@bridgecrew.io directly. Do not file a public issue. If
+English is not your first language, please try to describe the problem
+and its impact to the best of your ability. For greater detail, please
+use your native language and we will try our best to translate it using
+online services.
+
+Please also include the code you used to find the problem and the
+shortest amount of code necessary to reproduce it.
+
+Please do not disclose this to anyone else. We will retrieve a CVE
```

Comment:
we would accept pull requests on those like this: https://github.com/bridgecrewio/checkov/issues/381
how should that be mentioned here?